.ci/watchdog/Watchdog.py

```diff
@@ -151,10 +184,16 @@ def _check_prs(self, pull_requests):
                                         of github.PullRequest.PullRequest
         """
         current_prs = []
+        log.info('Reading ignore.json file in: {}'.format(_IGNORE_FILE_PATH))
+        file = open(_IGNORE_FILE_PATH, 'r')
```

Comment:
This will fail if `ignore.json` does not exist. Let's not fail here, but log a warning and continue working as if the file was empty.

Reply:
Fixed. Not reading ignore criteria from a file anymore.